### PR TITLE
[WM-1798] [WM-1800] Update nextstrain inputs/outputs to match tsv file

### PR DESCRIPTION
**Pull request by kpierre13**

### CODE REVIEW

**Review by THWiseman — APPROVED** [review]

File: service/src/main/resources/changelog/changesets/20230309_sarscov2_update_io_record_lookup_names.yaml [hunk 1]
```diff
@@ -0,0 +1,1490 @@
...
+                          "type": "primitive",
+                          "primitive_type": "String"
+                        },
+                        "source": {
+                          "type": "record_lookup",
+                          "record_attribute": "build_name"
+                        }
+                      },
+                      {
+                        "input_name": "sarscov2_nextstrain.builds_yaml",
```
Comment (salonishah11) [inline service/src/main/resources/changelog/changesets/20230309_sarscov2_update_io_record_lookup_names.yaml:26]: Should this and attribute name be `build_yaml`?

File: service/src/main/resources/changelog/changesets/20230309_sarscov2_update_io_record_lookup_names.yaml [hunk 2]
```diff
@@ -0,0 +1,1490 @@
...
+                    [
+                      {
+                        "input_name": "sarscov2_nextstrain.build_name",
+                        "input_type": {
+                          "type": "primitive",
+                          "primitive_type": "String"
+                        },
+                        "source": {
+                          "type": "record_lookup",
+                          "record_attribute": "build_name"
```
Comment (cjllanwarne) [inline service/src/main/resources/changelog/changesets/20230309_sarscov2_update_io_record_lookup_names.yaml:22]: 💯

File: service/src/main/resources/changelog/changesets/20230309_sarscov2_update_io_record_lookup_names.yaml [hunk 3]
```diff
@@ -0,0 +1,1490 @@
...
+                      },
+                      {
+                        "input_name": "sarscov2_nextstrain.builds_yaml",
+                        "input_type": {
+                          "type": "primitive",
+                          "primitive_type": "File"
+                        },
+                        "source": {
+                          "type": "record_lookup",
+                          "record_attribute": "builds_yaml"
```
Comment (cjllanwarne) [inline service/src/main/resources/changelog/changesets/20230309_sarscov2_update_io_record_lookup_names.yaml:33]: 💯

File: service/src/main/resources/changelog/changesets/20230309_sarscov2_update_io_record_lookup_names.yaml [hunk 4]
```diff
@@ -0,0 +1,1490 @@
...
+                                "type": "primitive",
+                                "primitive_type": "File"
+                              }
+                            },
+                            "non_empty": false
+                          }
+                        },
+                        "source": {
+                          "type": "record_lookup",
+                          "record_attribute": "assembly_fastas"
```
Comment (cjllanwarne) [inline service/src/main/resources/changelog/changesets/20230309_sarscov2_update_io_record_lookup_names.yaml:54]: 💯

File: service/src/main/resources/changelog/changesets/20230309_sarscov2_update_io_record_lookup_names.yaml [hunk 5]
```diff
@@ -0,0 +1,1490 @@
...
+                      },
+                      {
+                        "output_name": "sarscov2_nextstrain.counts_by_group",
+                        "output_type": {
+                          "type": "map",
+                          "key_type": "String",
+                          "value_type": {
+                            "type": "primitive",
+                            "primitive_type": "Int"
+                          }
```
Comment (cjllanwarne) [inline service/src/main/resources/changelog/changesets/20230309_sarscov2_update_io_record_lookup_names.yaml:1347]: 💯

**Review by cjllanwarne — APPROVED** [review]
👍  Looks great!

File: service/src/main/resources/changelog/changesets/20230309_sarscov2_update_io_record_lookup_names.yaml (hunk 1 above)
Comment (cjllanwarne) [reply service/src/main/resources/changelog/changesets/20230309_sarscov2_update_io_record_lookup_names.yaml:26]: The source of truth is the `nextstrain.tsv` data set in the [sample workspace](https://app.terra.bio/#workspaces/azure-featured-workspaces/COVID-19%20Surveillance%20/files), and the required workflow inputs in the [workflow script](https://raw.githubusercontent.com/broadinstitute/viral-pipelines/master/pipes/WDL/workflows/sarscov2_nextstrain.wdl). In this case, it's `build_name` and `builds_yaml`, so I think this is currently correct.

**Pull request merged**